kalamine/utils.py

```diff
@@ -47,15 +47,33 @@ class Layer(IntEnum):
     ALTGR = 4
     ALTGR_SHIFT = 5
 
+    @classmethod
+    def parse(cls, raw: str) -> Optional["Layer"]:
```

Comment:
As mentioned in another comment, I’m not a fan of having a `parse` method to be called after the object has been instantiated. To me it should either be a constructor, or we should have a factory that returns such an object.